recursion/tower_of_hanoi.go

```diff
@@ -0,0 +1,21 @@
+package recursion
+
+import "fmt"
+
+// https://en.wikipedia.org/wiki/Tower_of_Hanoi
+
```

Comment:
Please add a proper description (not just a link to a wikipedia) of the function immediately preceding the function definition for example:

```suggestion
// TowerOfHanoi <description goes here>
func TowerOfHanoi(n int, towerSource, towerDestination, towerAuxiliary string) {
```

This helps the automatic documentation generator parse the comments properly - you can see the documentation [here](https://pkg.go.dev/github.com/TheAlgorithms/Go). You can see the universal guidelines for commenting Go code and an example [here](https://golang.org/pkg/go/doc/#example_NewFromFiles)